debug_scripts/create-new-eeprom-with-self-signed-certs.sh

```diff
@@ -31,7 +31,9 @@ getEdgeStatus() {
 }
 
 readEeprom() {
-  deviceID=$(jq -r .deviceID ${IDENTITY_DIR}/identity.json)
+  if [ -f ${IDENTITY_DIR}/identity.json ] ; then
+    deviceID=$(jq -r .deviceID ${IDENTITY_DIR}/identity.json)
```

Comment:
Is deviceID suppose to have a default value if the file doesn't exist?

Reply:
Actually no, deviceID might not be present even if the identity.json is present.